src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java

```diff
@@ -33,13 +33,14 @@
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.CLAIMANT_ONLY;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.RESPONDENT_ONLY;
 import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.Helper.createLinkForUploadedDocument;
+import static uk.gov.hmcts.ethos.replacement.docmosis.service.TseService.NOT_STARTED_YET;
 
 @Service("sendNotificationService")
 @RequiredArgsConstructor
 @Slf4j
 @SuppressWarnings({"PMD.ExcessiveImports"})
 public class SendNotificationService {
-
+    public static final String NOT_VIEWED_YET = "notViewedYet";
```

Comment:
Is this more suitable for et-common?

Reply:
Not sure, another one I'm using is in callbacks NOT_STARTED_YET

Reply:
DOne